RFCs/platform-practice-4ps.md

```diff
@@ -0,0 +1,82 @@
+---
+name: Platform Practice 4Ps
+about: With this document, I am proposing a review of the platform practice using the 4Ps for meetings(Purpose, Product, Process and People)
+
+---
+
+<!-- Read the docs about [how to write an RFC at Artsy](https://github.com/artsy/README/blob/43c400d81ff9fee7276c3dd934de26b985da362f/playbooks/rfcs.md) before starting an RFC.
+ -->
+ 
+# Platform Practice
+
+Using the 4Ps format to organize our thoughts around this practice
+
+## Purpose:
+
+-   Disseminate good patterns: Share best practices and successful patterns within the platform development.
+    
+-   Actions to improve the platform: Identify and discuss actionable items to enhance the platform.
+    
+-   Avenue for people to bring what they find important: Provide a platform for individuals to contribute their insights and concerns.
+    
+-   Radiate intent of changes in the Platform: Communicate the purpose and intent behind any changes.
+    
+-   Evolving a platform towards a coherent direction: Ensure alignment among engineers and product teams, steering the platform in a cohesive direction.
+    
+
+## Product:
+
+*   Practice notes with the main topics: Document key points and decisions made during the practice.
+*   Possible action items discussed: Identify and outline potential actions for improvement.
+*   Information was shared and people aligned: Ensure information is disseminated and team members are aligned on the discussed topics.
+    
+
+## People:
+
+* It's open to everybody in PDDE and part of the engineering team calendar: Encourage inclusivity by making the practice accessible to all team members.
+
+### Roles of this practice:
+
+#### 1. Facilitator:
+   * The Facilitator plays a crucial role in managing the process of the platform practice. This includes time-keeping, organizing the agenda, reviewing takeaways and action items, and diligently documenting sessions in Notion. The Facilitator ensures the smooth flow and effectiveness of each session.
```

Comment:
> diligently documenting

I might loosen this to documenting decisions or follow-ups. It's possible we've been _too_ diligent, noting everything discussed at the expense of engaging fully in the discussion.

Reply:
Agreed, I changed the description.